CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Change: [#1623] Title screen music volume is now bound to the music volume setting.
 - Fix: [#1237] Long entity (company) names may be cut-off incorrectly.
 - Fix: [#1650] Acquire all company assets cheat may cause some trains and trams to crash.
+- Fix: [#1608] Added character limit label in text input windows.
```

Comment:
This warrants being marked a feature rather than a (bug)fix, I think. Could you move it to the top as well? (Features go above changes.)

Reply:
Sorry, didn't mean to mark it as a fix. I'll change it in a second.